src/test/java/Homework22.java

```diff
@@ -0,0 +1,20 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework22 extends BaseTest {
+
+
+    @Test
+    public void renamePlaylist() {
+        String newPlaylistName = "Manasa Intellij";
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)